components/WrapperCanvas.vue

```diff
@@ -185,11 +185,13 @@
         } else {
             height = setToRelative(200, currentHeight.value);
             width = setToRelative(200, currentWidth.value);
-            name = 'Double-click to edit me.';
```

Comment:
I am not sure if it is good to fully delete this hint, since how are the users than supposed to know how to enter the text-edit-process. Although I would agree that it might be misleading, when the editor is already open.

Reply:
Maybe it would make sense adding the "Double-click to edit me." back in if it otherwise would be an empty string. This would still not solve the issue though, if the user would add a text when to pop-up opens, since how would they than be supposed to know how to edit it again.